Build.mak

```diff
@@ -1,6 +1,3 @@
-# This will make D2 unittests fail if stomping prevention is triggered
-export ASSERT_ON_STOMPING_PREVENTION=1
-
```

Comment:
I have restored this bit, at least until D2 `-cov -inline` bug is fixed and we can test that too